extensions/wikia/CreatePage/CreatePage.i18n.php

```diff
@@ -16,7 +16,7 @@
 	'createpage_cant_edit' => 'Could not perform edit',
 	'createpage-dialog-title' => 'Add New Page',
 	'createpage-dialog-message2' => 'Write your page title here:',
-	'createpage-dialog-message3' => 'We currently have {{NUMBEROFARTICLES}} pages on {{SITENAME}}. We need to check if the name is available.',
+	'createpage-dialog-message3' => 'We currently have {{NUMBEROFARTICLES}} {{PLURAL:$1|page|pages}} on {{SITENAME}}. We need to check if the name is available.',
```

Comment:
Are we sure that by doing that we won't break all existing translations in crowding when we upload that file there?